### PR TITLE
Normalize special page aliases

### PR DESCRIPTION
**Pull request by arlolra**
/cc @wikimedia/services @subbuss

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -294,6 +296,19 @@ function _checkMaxLength(title, namespace) {
     }
 }
 
+function _normalizeSpecialPage(title, siteInfo) {
+    var parts = title.split('/');
+    var first = parts[0];
+    var alias = arrayFind(siteInfo.specialpagealiases, function(o) {
```
Comment (Pchelolo) [inline lib/index.js]: This makes it a breaking change for old clients - they wouldn't provide the `specialpagealiases` key in the siteInfo and normalization will fail for special pages. So we need to either do a major version bump or add a check that `spacialpagealiases` and present indeed.

Comment (arlolra) [inline lib/index.js]: Would you accept `siteInfo.specialpagealiases || []`?

Comment (Pchelolo) [inline lib/index.js]: Yup, I think that's better, just need to remember to update the clients

**Comment by arlolra** [comment]
Oh wait, I should update the README as well for siteInfo.

**Comment by arlolra** [comment]
> Oh wait, I should update the README as well for siteInfo.

Ok, good to go.

**Pull request merged**